lib/open_food_network/variant_stock.rb

```diff
@@ -22,39 +34,65 @@ def on_hand
       end
     end
 
+    # Alias for Spree's 2.0 #total_on_hand
+    #
+    # @return [Float|Integer]
     def count_on_hand
       total_on_hand
     end
 
+    # Sets the stock level when `track_inventory_levels` config is
+    # set. It raises otherwise.
+    #
+    # @raise [StandardError] when the track_inventory_levels config
+    # key is not set.
     def on_hand=(new_level)
       error = 'Cannot set on_hand value when Spree::Config[:track_inventory_levels] is false'
       raise error unless Spree::Config.track_inventory_levels
 
       self.count_on_hand = new_level
     end
 
+    # Sets the stock level. As opposed to #on_hand= it does not check `track_inventory_levels`'s value. It does ensure there's a stock item for the variant however raising otherwise. See #raise_error_if_no_stock_item_available for details.
```

Comment:
More then explaining what's already in the code, I'd prefer to see here the why it is doing so. Why is it not checking "track_inventory_levels" here? same for on_hand=, why is it not checking for the existence of stock_item?